app/views/devise/registrations/edit.html.haml

```diff
@@ -0,0 +1,6 @@
+/ NOT USED: users should not be able to sign up themselves.
+/ Default devise view can be regenerated if needed using
+/ `rails generate devise:views -v registrations`
+/ but if you do so ensure the users/registrations_controller.rb
+/ prevents sign up by individuals.
+/
```

Comment:
Looks like we do may need a registrations edits (possibly) and registrations new

look in the registrations controller - but to cut a long story short, only on the api-sandbox where we allow vendors to register themselves